homeassistant/components/tado/repairs.py

```diff
@@ -13,19 +13,19 @@
 
 def manage_water_heater_fallback_issue(
     hass: HomeAssistant,
-    water_heater_entities: list,
+    water_heater_names: list,
```

Comment:
If we know that these are strings it's good to type that.
```suggestion
    water_heater_names: list[str],
```